Vates/VatesAPI/inc/MantidVatesAPI/PresenterFactories.h

```diff
@@ -13,15 +13,15 @@ class WorkspaceProvider;
 
 class DLLExport EmptyWorkspaceNamePolicy {
 protected:
-  std::string getWorkspaceName(Mantid::API::IMDWorkspace_sptr) {
-    return "__EmptyWorkspaceNamePolicy";
-  }
+  const std::string &
+  getWorkspaceName(const Mantid::API::IMDWorkspace & /*workspace*/);
```

Comment:
I thought we were trying to remove the comments from argument names in these cases, but I guess there is no project-wide policy on this.